nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment-api/src/main/java/org/nuxeo/ecm/platform/comment/api/CommentManager.java

```diff
@@ -186,6 +188,17 @@
      */
     List<Comment> getComments(CoreSession session, String documentId);
 
+    /**
+     * Gets all comments for a document.
+     *
+     * @param session the core session
+     * @param documentId the document id
+     * @param sortAscending whether to sort ascending or descending
+     * @return the list of comments, or an empty list if no comment is found
+     * @since 10.3
+     */
+    List<Comment> getComments(CoreSession session, String documentId, boolean sortAscending);
```

Comment:
What the purpose of `sortAscending` ? Comments should already by sorted in one way, if you need them to be reverse sorted can't you do it client side ?
My concern is this service already has a lot of API and I don't see this one as needed.

Reply:
JSF UI and WebUI use different sort orders (ascending and descending, respectively)